flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -49,6 +62,23 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler {
     channel.setMethodCallHandler(null)
   }
 
+  override fun onAttachedToActivity(binding: ActivityPluginBinding) {
+    activity = WeakReference(binding.activity)
+  }
+
+  override fun onDetachedFromActivity() {
+    activity = null
+    framesTracker = null
+  }
+
+  override fun onReattachedToActivityForConfigChanges(binding: ActivityPluginBinding) {
+    // Stub
+  }
+
+  override fun onDetachedFromActivityForConfigChanges() {
+    // Stub
+  }
```

Comment:
l: Are we required to override this function even though we have no code to add? Can we just delete this empty functions?

Reply:
Unless methods have a default signature, you have to override.

Reply:
Ow, because we are implementing an interface and not inheriting a class, got it. 
Im spending too much time with objc, lol.